llama_stack/providers/inline/vector_io/faiss/faiss.py

```diff
@@ -41,6 +45,7 @@
 VECTOR_DBS_PREFIX = f"vector_dbs:{VERSION}::"
 FAISS_INDEX_PREFIX = f"faiss_index:{VERSION}::"
 OPENAI_VECTOR_STORES_PREFIX = f"openai_vector_stores:{VERSION}::"
+VECTOR_DIMENSION = 128  # sample dimension
```

Comment:
Since we only use this for `health()` and very specific I think this should be part of `health()` only.